router/auth.go

```diff
@@ -72,11 +72,18 @@ func authPassThrough(log *log.Logger, w http.ResponseWriter, req *http.Request,
 		return authInjectYellowfin(log, w, req, authData, target)
 	case AuthPassThroughSitePro:
 		return authInjectSitePro(log, w, req, target)
+	case AuthPassThroughECS:
+		return authInjectECS(log, w, req, target)
 	default:
 		return true
 	}
 }
 
+func authInjectECS(log *log.Logger, w http.ResponseWriter, req *http.Request, target *targetPassThroughAuth) bool {
+	req.SetBasicAuth(target.config.Username, target.config.Password)
```

Comment:
@LeonLeibbrandt  we can combine ECS and SitePro auth into `authInjectBasicAuth` because methods are identical